src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1284,6 +1284,15 @@ ApplicationManagerImpl::GetHandshakeContext(uint32_t key) const {
   return SSLContext::HandshakeContext();
 }
 
+bool ApplicationManagerImpl::IsNaviApp(const int32_t& session_key) const {
+  ApplicationSharedPtr app = application(session_key);
```

Comment:
@tkireva , `session_key`? Think `connection_key` must be here, which takes from connection_handle and session_key